.github/workflows/_unit-tests-and-static-checks.yml

```diff
@@ -45,4 +50,16 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: -- --skip clique_network
+          args: --workspace --exclude baby-liminal-extension -- --skip clique_network
+
+      # Run unit tests for the chain extension: they require a different setup.
+
+      - name: Install Nightly Rust Toolchain
+        uses: Cardinal-Cryptography/github-actions/install-rust-toolchain@v6
+        with:
+          channel: nightly-2023-05-22-unknown-linux-gnu
+          targets: wasm32-unknown-unknown
+
+      - name: Run unit tests for chain extension
```

Comment:
Let's add this step as a parallel job - we have a bunch of large runners to use and adding this as a step here increases the whole PR pipeline by 3-4 minutes (https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/7912430437?pr=1614), where's baseline is that whole `.github/workflows/_unit-tests-and-static-checks.yml` runs now around 8.5 minutes https://github.com/Cardinal-Cryptography/aleph-node/actions/runs/7917287724

Reply:
done